web/index.html

```diff
@@ -159,7 +159,7 @@
           <linearGradient href="#a" id="b" x1="-2996.72" x2="-1559.018" y1="4654.921" y2="4200.613" gradientTransform="matrix(.09408 0 0 .09408 -16454.439 2288.328)" gradientUnits="userSpaceOnUse" />
           <linearGradient href="#a" id="c" x1="-12403.771" x2="-12550.984" y1="16503.461" y2="9437.118" gradientTransform="matrix(.0249 0 0 .0249 -16447.602 2300.046)" gradientUnits="userSpaceOnUse" />
         </defs>
-        <g stroke-width="2.058" aria-label="nsect" fill="currentColor">
+        <g stroke-width="2.058" aria-label="insect" fill="currentColor">
```

Comment:
I'm not too sure about this and the one below, are they intentional?

Reply:
During the design process, I probably wrote "nsect" into a text field, because the "i" was designed separtely(?). Not sure. Maybe let's just leave this file untouched. Or remove the label? What is the purpose of `aria-label` anyway?

Reply:
Here's an explanation: https://stackoverflow.com/a/22040485

Reply:
Ah, okay. So let's leave your changes in. Thanks!